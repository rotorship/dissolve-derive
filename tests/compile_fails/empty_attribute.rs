use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct EmptyAttribute {
	#[dissolved]
	field: String,
}

fn main() {}
