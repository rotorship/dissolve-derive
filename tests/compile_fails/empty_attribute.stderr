error: dissolved attribute requires options, use #[dissolved(skip)] or #[dissolved(rename = "new_name")] instead
 --> tests/compile_fails/empty_attribute.rs:5:2
  |
5 |     #[dissolved]
  |     ^^^^^^^^^^^^
